orm/ai-sdk-nextjs/README.md

```
@@ -0,0 +1,94 @@
+
+2. Create a new API key and give it full access
+
+3. Add the API key to the `.env`.
+
+### 5. Start the development server
+
+```
+npm run dev
+```
+
+The server is now running at http://localhost:3000
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)